gromacs/testing/nosetester.py

```diff
@@ -0,0 +1,218 @@
+"""
+Nose test running.
```

Comment:
No, ripped out and adapted from pandas. I assumed nose will need a custom NoseTester class but maybe that's not needed actually - i'll check it out and get back to you on this